libs/common/src/lib/inject-map-api.service.ts

```diff
@@ -0,0 +1,40 @@
+import { Injectable } from '@angular/core';
+import { SettingsService } from '@placeos/common';
+import { BehaviorSubject, Observable, Subject } from 'rxjs';
+
+@Injectable({
+    providedIn: 'root',
+})
+export class InjectMapApiService {
+    private maps_key_exists = new BehaviorSubject<boolean>(null);
+
+    private _setKeyExists(value: boolean) {
```

Comment:
I don't think you need a function for one line of code. Especially if it is private.

Reply:
I've now removed the function and set the BehaviorSubject value directly.